server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -579,7 +589,7 @@ void createIndex(String index, boolean requireDataStream, TimeValue timeout, Act
     void rolloverDataStream(String dataStream, TimeValue timeout, ActionListener<RolloverResponse> listener) {
```

Comment:
Super minor, but should we rename this to `autoRolloverDataStream` to indicate that it's only for automatic rollovers?